src/auth/webhook-secret.guard.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  CanActivate,
+  ExecutionContext,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
```

Comment:
PullProofAI

Remove unused imports to keep the code clean and improve readability.
```diff
- import * as crypto from 'crypto';
- import { Request } from 'express';
```

#### Issue: Inconsistent Naming


<!-- This is an auto-generated comment by PullProofAI -->